src/queries/get-cases-by-id.js

```diff
@@ -6,16 +6,13 @@ const SUCCESS_CODES = [200];
 async function getCasesById(casesIds, options) {
   const Logger = getLogger();
 
-  if (casesIds.length === 0) {
-    Logger.trace('No cases found for the given indicators.');
-    return {};
-  }
-
   const fields = ['tags', 'attributes'];
+  const tql = casesIds.length > 0 ? `id IN (${casesIds.join(',')})` : '';
```

Comment:
What happens here if `caseIds` is empty?  Does this endpoint then just return all cases?  What is the scenario where we want a call to `getCasesById` to run even if no case IDs are provided?